src/charts/line.js

```diff
@@ -598,7 +598,7 @@ define(function(require){
 
                 let newDates = dates.map(d => ({
                        date: new Date(d[dateLabel]),
-                       value: +d[valueLabel]
+                       value: d[valueLabel] === null ? null : +d[valueLabel]
```

Comment:
Do you mind creating a variable above `newDates` and assign this little logic to that variable?
```
const value = d[valueLabel] === null ? null : +d[valueLabel];
...
({
date: new Date(d[dateLabel]).
value
})
```

Reply:
d is only defined within the map.
I don't see the advantage but I can do:
```
                let newDates = dates.map(d => {
                    const value = d[valueLabel] === null ? null : +d[valueLabel];
                    return {
                        date: new Date(d[dateLabel]),
                        value
                    };
                });
```

Reply:
What if we have an 'acceptNullValue' function?

const acceptNullValue = (value) => value === null ? null : value;

Reply:
we still need to + the value to produce a valid number

Reply:
Oh, yeah, let's add that too

Reply:
done